errors/runtime-css-import.md

```diff
@@ -0,0 +1,63 @@
+# Runtime CSS import
+
+#### Why This Error Occurred
+
+Runtime CSS imports are bad for perfomance because the browser can't know about the `@import` until the stylesheet is downloaded. This means the stylesheets can't be fetched in parallel.
+
+#### Possible Ways to Fix It
+
+Import the CSS file in a JavaScript module or use a relative `@import` in your CSS.
```

Comment:
I think we should also point towards using a link element

Reply:
Second example recommends link element for remote stylesheets 👍